config/defaults.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
```

Comment:
Ah, right -- I'm planning to refactor the cli script and its handling of the defaults in a separate PR.